clusterloader2/testing/watch-list/deployment.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{.Name}}
+  labels:
+    group: watch-list
+spec:
+  replicas: {{.Replicas}}
+  selector:
+    matchLabels:
+      group: watch-list
+  template:
+    metadata:
+      labels:
+        group: watch-list
+    spec:
+      containers:
+      # TODO: change the image ref
+      - image: quay.io/polynomial/simple-watch:latest
```

Comment:
the actual test is defined here https://github.com/kubernetes/perf-tests/pull/2256